fhir/resources/DSTU2/visionprescription.py

```diff
@@ -0,0 +1,218 @@
+# -*- coding: utf-8 -*-
+"""
+Profile: https://www.hl7.org/fhir/DSTU2/visionprescription.html
+Release: DSTU2
+Version: 1.0.2
+Revision: 7202
+"""
+from typing import List as ListType
+
+from pydantic import Field
+
+from . import domainresource, fhirtypes
+from .backboneelement import BackboneElement
+
+
+class VisionPrescription(domainresource.DomainResource):
+    """Prescription for vision correction products for a patient.
+
+    An authorization for the supply of glasses and/or contact lenses to a patient.
+    """
+
+    resource_type = Field("VisionPrescription", const=True)
+
+    identifier: ListType[fhirtypes.IdentifierType] = Field(
+        None,
+        alias="identifier",
+        title="Business Identifier for the resource",
+        description="A unique identifier assigned to this explanation of benefit.",
+    )
+
+    datewritten: fhirtypes.DateTime = Field(
```

Comment:
Mismatch case: should be `dateWritten`

Reply:
fixed and updated the PR. thanks for catching.